Themes/default/PersonalMessage.template.php

```diff
@@ -495,8 +496,8 @@ function template_single_pm($message)
 						<span class="smalltext">&#171; ', $context['folder'] == 'sent' ? $txt['pm_sent_is_replied_to'] : $txt['pm_is_replied_to'], ' &#187;</span>';
```

Comment:
Please also remove all the useless `&#171;` and `&#187;` entities in this file. These ones escaped my notice when I was hunting down and killing the rest of them in #7261. 🔪💣

Reply:
Yessirrr